src/highdicom/ann/sop.py

```diff
@@ -98,6 +102,21 @@ def __init__(
             of `highdicom.base.SOPClass`
 
         """  # noqa: E501
+        coordinate_type = AnnotationCoordinateTypeValues(
+            annotation_coordinate_type
+        )
+        if len(set([img.FrameOfReferenceUID for img in source_images])) > 1:
```

Comment:
Set comprehension :o

```suggestion
        if len({img.FrameOfReferenceUID for img in source_images}) > 1:
```